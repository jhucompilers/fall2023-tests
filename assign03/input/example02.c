int add(int a, int b);

int main(void) {
  int sum;
  sum = add(1, 2);
  return sum;
}
