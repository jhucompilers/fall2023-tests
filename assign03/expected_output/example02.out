0|add|function|function (int, int) returning int
0|main|function|function () returning int
2|sum|variable|int
